### PR TITLE
Concise items

### PR DESCRIPTION
**Pull request by sumeetadur**
Provides the option for displaying concise items in the stream. Concise items are displayed without a tag and subheading.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "n-message-prompts": "Financial-Times/n-message-prompts#^v2.0.0",
     "n-notification": "^2.0.0",
     "n-topic": "^5.0.0",
-    "n-barriers": "^1.0.0"
+    "n-barriers": "^1.0.0",
+    "o-colors": "^3.3.1"
```
Comment (andygnewman) [inline bower.json]: You get o-colors with next-sass-setup, which is pulled in by n-layout

Comment (sumeetadur) [inline bower.json]: Thanks!

**Comment by sumeetadur** [comment]
@matthew-andrews could you have a look through this please? Thanks!

**Comment by callumlocke** [comment]
this work is now in #904

**Pull request closed without merging**